db/migrate/20230609065335_create_events.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEvents < ActiveRecord::Migration[7.0]
+  def change
+    create_table :events do |t|
+      t.string :title
+      t.text :description
+      t.datetime :eventdate
```

Comment:
For better practice, please use `event_date` to differentiate 2 words

Reply:
I was originally using event_date but then it would only store date in db and not time even though I was using datetime, that's why I shifted to eventdate.